.gitignore

```
@@ -208,6 +208,13 @@ rel/
 mambaforge/
 miniconda3/
 
+# in source build of conda pacakges
+conda/recipes/channeldata.json
+conda/recipes/index.html
+conda/recipes/linux-64/
+conda/recipes/noarch/
+conda/recipes/tmp/
```

```suggestion
conda/recipes/linux-64/
conda/recipes/noarch/
conda/recipes/osx-64/
conda/recipes/tmp/
conda/recipes/win-64/
```
We need to add the directories for people building on Windows and macOS too.